### ISSUE TITLE
Pressing D will disable the JS script but won't show anything

### ISSUE THREAD

**Issue by JeFawk** [Scirra/Construct-3-bugs#3266]
## Problem description

The shortcut to disable a block/action/condition works for scripts but doesn't have any feedback

## Steps to reproduce

1. open an empty project
2. make a hello world script on start of layout
3. select the script, press D
4. run the project

## Observed result

Script won't run

## More details

**Affected browsers/platforms:** Chrome

**First affected release:** r 157

## System details

<details><summary>View details</summary>

Platform information
Browser: Chrome
Browser version: 76.0.3809.100
Browser engine: Chromium
Browser architecture: 64-bit
Context: browser
Operating system: Windows
Operating system version: 10
Operating system architecture: 64-bit
Device type: desktop
Device pixel ratio: 1.5
Logical CPU cores: 4
Approx. device memory: 8 GB
User agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/76.0.3809.100 Safari/537.36
C3 release: r157 (stable)
Language setting: en-US

Local storage
Storage quota (approx): 170 gb
Storage usage (approx): 420 mb (0.2%)
Persistant storage: No

Browser support notes
This list contains missing features that are not required, but could improve performance or user experience if supported.

Nothing is missing. Everything is OK!
WebGL information
Version string: WebGL 2.0 (OpenGL ES 3.0 Chromium)
Numeric version: 2
Supports NPOT textures: yes
Supports GPU profiling: yes
Supports highp precision: yes
Vendor: Google Inc.
Renderer: ANGLE (Radeon 550 Series Direct3D11 vs_5_0 ps_5_0)
Major performance caveat: no
Maximum texture size: 16384
Point size range: 1 to 1024
Extensions:

EXT_color_buffer_float
EXT_disjoint_timer_query_webgl2
EXT_float_blend
EXT_texture_filter_anisotropic
KHR_parallel_shader_compile
OES_texture_float_linear
WEBGL_compressed_texture_s3tc
WEBGL_compressed_texture_s3tc_srgb
WEBGL_debug_renderer_info
WEBGL_debug_shaders
WEBGL_lose_context
Audio information
System sample rate: 48000 Hz
Output channels: 2
Output interpretation: speakers
Supported decode formats:

Ogg Opus (audio/ogg; codecs=opus)
WebM Opus (audio/webm; codecs=opus)
Ogg Vorbis (audio/ogg; codecs=vorbis)
WebM Vorbis (audio/webm; codecs=vorbis)
MPEG-4 AAC (audio/mp4; codecs=mp4a.40.5)
MP3 (audio/mpeg)
FLAC (audio/flac)
PCM WAV (audio/wav; codecs=1)
Supported encode formats:

WebM Opus (audio/webm; codecs=opus)
Video information
Supported decode formats:

WebM VP9 (video/webm; codecs=vp9)
WebM VP8 (video/webm; codecs=vp8)
Ogg Theora (video/ogg; codecs=theora)
H.264 (video/mp4; codecs=avc1.42E01E)
Supported encode formats:

WebM VP9 (video/webm; codecs=vp9)
WebM VP8 (video/webm; codecs=vp8)

</details>

**Comment by AshleyScirra** [comment]
Seems to work fine here. Please don't skimp on bug reports, it makes it hard to follow. For example it's not clear if you're using a script file, block, or action; step 3 implies it's not a script file; I tried a script block and it appears to work fine (adds strikethrough style, script doesn't run when previewing). It's not clear what you're actually seeing or what you expected to see instead. The observed result is also the correct result given what you've done.
